### DISCUSSION TITLE
Game consistently crashes after about 30-45 seconds

### DISCUSSION THREAD

**Discussion by bmacle03** [ppy/osu#16035] [Q&A] [answered]
[Logs.zip](https://github.com/ppy/osu/files/7695143/Logs.zip)
Not sure if this is important but I am on windows 11.

**Comment by bdach** [comment, marked as answer]
attached log is from stable, this repository is for lazer only. please go to https://github.com/ppy/osu-stable-issues/ to report this instead

**Reply by bmacle03** [reply]
ooops my bad. ty
